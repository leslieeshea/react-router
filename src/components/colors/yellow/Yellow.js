import React from 'react';

export default function Yellow() {
  const style = {
    backgroundColor: 'yellow',
    width: '50vh',
    height: '50vh'
  };

  return <div style={style}></div>;
}
