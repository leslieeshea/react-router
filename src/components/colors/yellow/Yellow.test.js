import React from 'react';
import { shallow } from 'enzyme';
import Yellow from './Yellow';

describe('Yellow component', () => {
  it('renders a yellow splotch', () => {
    const wrapper = shallow(<Yellow />);
    expect(wrapper).toMatchSnapshot();
  });
});
