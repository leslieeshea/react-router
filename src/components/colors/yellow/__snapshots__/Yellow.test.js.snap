// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Yellow component renders a yellow splotch 1`] = `
<div
  style={
    Object {
      "backgroundColor": "yellow",
      "height": "50vh",
      "width": "50vh",
    }
  }
/>
`;
